### ISSUE TITLE
customFields with elastic does not seem to work

### ISSUE THREAD

**Issue by cherez0ff** [fjogeleit/policy-reporter#217]
Hi. I can't enable customFields for elasticsearch.

I used policy-reporter-2.13.2
Below is a part of values for elasticsearch and for slack, and messages sent to them. Elastic message does not contain a custom field, but Slack message does.

I double-checked that same config values are in k8s, and version of deployed pod is ghcr.io/kyverno/policy-reporter:2.10.1

```yaml
elasticsearch:
  certificate: '/var/tmp/cacert.crt'
  customFields:
    - cluster: 'cluster1'
  host: "***"
  index: "sec-kyverno"
  username: "***"
  password: "***"
  rotation: "daily"
  minimumPriority: "warning"
  skipExistingOnStartup: true
slack:
  webhook: "***"
  customFields:
    - cluster: 'cluster1'
  minimumPriority: "warning"
  skipExistingOnStartup: true
```

```json
{
  "_index": "sec-kyverno-2022.10.25",
  "_type": "event",
  "_id": "4fW1DoQBY5GgZkUiFVgc",
  "_version": 1,
  "_score": null,
  "_source": {
    "Message": "Running as root is not allowed. The fields spec.securityContext.runAsUser, spec.containers[*].securityContext.runAsUser, spec.initContainers[*].securityContext.runAsUser, and spec.ephemeralContainers[*].securityContext.runAsUser must be  set to a number greater than zero ",
    "Policy": "require-no-container-root",
    "Rule": "no-run-as-root-user",
    "Status": "fail",
    "Severity": "high",
    "Category": "Pod Security Standards (Restricted)",
    "Source": "Kyverno Event",
    "Timestamp": "2022-10-25T10:36:57Z",
    "Properties": {
      "eventName": "require-no-container-root.172144711e9ed465",
      "time": "2022-10-25T10:36:57Z"
    },
    "Resource": {
      "APIVersion": "",
      "Kind": "Pod",
      "Name": "test3-1337",
      "Namespace": "1337",
      "UID": ""
    },
    "Priority": "error",
    "Scored": false
  },
  "fields": {
    "Timestamp": [
      "2022-10-25T10:36:57.000Z"
    ]
  },
  "sort": [
    1666694217000
  ]
}
```
![image](https://user-images.githubusercontent.com/63862323/197752085-f56756e3-3a51-4ee5-984d-2b25a6c93185.png)

**Comment by cherez0ff** [comment]
Small mistake: I meant "cluster1337" instead of "cluster1" in config (values.yaml) part. Everything else is the same as in the latest test

**Comment by fjogeleit** [comment]
thanks, I will check what I missed

**Comment by fjogeleit** [comment]
As I see it's an mistake in the Helm Chart. It didn't update the version tag properly. Can you change the `image.tag` to `2.10.2` and try it again?

I will create a fixed release today

**Comment by cherez0ff** [comment]
2.10.2 does not seem to exist

https://github.com/orgs/kyverno/packages/container/package/policy-reporter

⇒ docker run -it --rm ghcr.io/kyverno/policy-reporter:2.10.2
Unable to find image 'ghcr.io/kyverno/policy-reporter:2.10.2' locally
docker: Error response from daemon: manifest unknown.
See 'docker run --help'.

**Comment by cherez0ff** [comment]
Sorry about that open-reopen, misclicked

**Comment by fjogeleit** [comment]
hm you right ... my mistake ... let me create the version.

**Comment by cherez0ff** [comment]
Also, I noticed that at the same time (in different locations) used customField and customFields (last letter difference). Specifically in default values.yaml used customField, and (almost?) everywhere else used customFields
May also be unwanted.

```bash
git clone https://github.com/kyverno/policy-reporter.git
cd policy-reporter
grep -iR customField | grep -v 'customFields' 
grep -iR 'customField'
```

**Comment by fjogeleit** [comment]
I will check that as well

**Comment by fjogeleit** [comment]
I released `v2.13.4` of the Helm Chart with a fixed app version
